env.go

```diff
@@ -17,3 +22,26 @@ func (e *Env) Getenv(key string, def ...string) string {
 
 	return val
 }
+
+// GetenvFile - Also checks file
+func (e *Env) GetenvFile(fs vfs.Filesystem, key, def string) string {
```

Comment:
Ok, it's in https://github.com/hairyhenderson/gomplate/pull/181 - I made a change to the approach - instead of having a separate function, the functionality is built-in to the existing `Getenv` function. This makes it easier to use in a template context, since the template author doesn't need to know how the user is going to provide the data.